src/StructDefinitionStatementNode.cpp

```diff
@@ -44,6 +45,54 @@ namespace cmm
         return blockNode.size();
     }
 
+    std::optional<std::string> StructDefinitionStatementNode::setupFieldTable()
+    {
+        // Only re/calculate if the fieldMap is empty.
+        if (!fieldMap.empty())
+        {
+            return std::nullopt;
+        }
+
+        u32 index = 0;
```

Comment:
Field class and/or IField use s32 so that they can use '-1' to indicate "not yet defined".  Consider making this a s32 for consistency.

Reply:
Resolved in commit 'f7899b3bcb6b2f7f852dcc3bcfab21407dc03d54'.